### ISSUE TITLE
Check-out process/ZIP Finder

### ISSUE THREAD

**Issue by vcfolger** [vcfolger/southernicecreamtx.com#83]
So now that the ZIP finder is up and running, I tested a couple of ZIPs. It seems to be working :-)

However, the result should tell the person what fee they have to pay, not just whether or not they are in the service area. Otherwise, the user has no way to know what to enter when they get to the "Extended Service Area Fee" question.

Also I don't understand the shipping/ship to a different address option in the checkout screen.

![screen shot 2016-11-21 at 11 29 10 pm](https://cloud.githubusercontent.com/assets/20050245/20512014/5f18e20c-b042-11e6-8dbd-097986248906.png)

**Comment by vcfolger** [comment]
Josh, here is what I have for the ZIP code finder results. Also note that we still need to update copy in the finder to capitalize ZIP.

![image](https://cloud.githubusercontent.com/assets/20050245/20845476/e000ecc0-b88a-11e6-87d8-a0b9ea15dc76.png)

*There should actually be four possible results:
Sorry...
 ...$50 
...$100
...$150 ($150 is an additional category of ZIP codes that was added when they submitted the ZIP list to me--you have it in there it just wasn't in the copy deck)

**Comment by vcfolger** [comment]
Josh, let me know if you need clarification on this

**Comment by vcfolger** [comment]
![screen shot 2017-01-13 at 8 38 27 am](https://cloud.githubusercontent.com/assets/20050245/21933648/ca786a66-d96b-11e6-8de7-2f33d12dedfe.png)

I don't see updated wording in the result field and zip in green box above is still lowercase (needs to be ZIP code)

**Comment by joshbolinger** [comment]
Ah, I changed the message somewhere else. I cant easily change this message http://cl.bolinger.co/ikxk . 

We have 2 different services running, 1 is basically an extension of WooCommerce that adds a shipping fee based on ZIP. The other checks to make sure your ZIP is on the list, it feeds back a YES or NO thats what goes here (http://cl.bolinger.co/ikxk) and here (http://cl.bolinger.co/ikvQ). This second one is not connected to the checkout process and does not have dynamic values for separating the different costs per ZIP.

Anyway, long story short, this would take me a few days to build this functionality

**Comment by vcfolger** [comment]
I thought it might be something like that. OK. Then we just need to change the message in teal above the finder result. It currently says that the user will see what their fee will be. 

Change to: 
First, enter your ZIP code to determine if we can  service your location.. 
An additional charge will be added at checkout if your location is outside of our primary service area.

**Comment by joshbolinger** [comment]
it breaks into 3 lines now, want to modify?

http://cl.bolinger.co/il0W

**Comment by vcfolger** [comment]
Yes, can we add a line break  before the second sentence and make the font a couple points smaller on the second sentence as well?
